vm/actor_interface/src/builtin/cron/mod.rs

```diff
@@ -1,16 +1,16 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+use address::Address;
 use ipld_blockstore::BlockStore;
 use serde::Serialize;
 use vm::ActorState;
 
 /// Cron actor address.
-pub static ADDRESS: &fil_actors_runtime_v7::builtin::singletons::CRON_ACTOR_ADDR =
-    &fil_actors_runtime_v7::builtin::singletons::CRON_ACTOR_ADDR;
+pub const ADDRESS: Address = Address::new_id(3);
```

Comment:
Not reliably, no. The runtime crate only defines the ids that they use internally and that set may change at any point. It would be nice to keep all of the magic values together, though, and not spread out over multiple modules.